pkg/eventemitter/eventemitter.go

```diff
@@ -400,6 +415,14 @@ func newEventHandlerKey(kindNamespaceName string, handlerType string) string { /
 	return fmt.Sprintf("%s.%s", kindNamespaceName, handlerType)
 }
 
+func getPrefixIdentifierFromKey(handlerKey string) string {
```

Comment:
Let's add a comment about this function. It can be confusing in the future not having the context in mind